Encountify/Views/RegisterPage.xaml.cs

```diff
@@ -73,8 +105,38 @@ private Boolean ValidUsername(string username)
 
         private Boolean ValidPassword(string password)
         {
-            if (string.IsNullOrEmpty(password))
+            var passwordValidation = Password.Text;
+            
+            var hasUpperCase = new Regex(@"[A-Z]+");
+            var hasLowerCase = new Regex(@"[a-z]+");
+            var hasNumbers = new Regex(@"[0-9]+");
+            var hasSpecialSymbols = new Regex(@"[!@#$%^&8()_+=\[{\]};:<>|.?,-]");
+            var hasRequiredLength = new Regex(@".{8,15}");
+
+
+            if (!hasUpperCase.IsMatch(passwordValidation))
             {
+                DisplayAlert("Error", "Password should contain at least one upper case letter", "Cancel");
```

Comment:
Invalid email and invalid password alerts looks different. I believe, that it would be a better design choice to make both similar for more unified look.
![Password](https://user-images.githubusercontent.com/50084966/135767332-031e722f-c81c-41d0-b45f-68679ae4a903.PNG)
![Email](https://user-images.githubusercontent.com/50084966/135767333-43ca7be0-5d79-4c62-b200-3e2e8accdd69.PNG)